src/components/Content/Items/Items.js

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import { Well } from "react-bootstrap";
+
+const joinClasses = (class1String, class2String) => (class1String || "") + (class2String ? " " + class2String : "");
```

Comment:
Use classnames for this stuff import classNames from "classnames"

Reply:
`classNames("eui-items", containerClassName)` will do the job

Reply:
Right, it works. Fixed